README.md

```diff
@@ -290,52 +290,52 @@ Before deploying, make sure you have installed Terraform and a Docker client (e.
 * Login to glcoud
 
 ```bash
-$ gcloud auth application-default login
```

Comment:
I'm fine with these changes but can you do them in another pr

Reply:
moved to #1352